README.md

```diff
@@ -515,6 +516,51 @@ strategies:
          - "Pending"
 ```
 
+### RemoveFailedPods
+
+This strategy evicts pods that are in failed status phase.
```

Comment:
This could describe a lot more of what the strategy does, especially the parameters